lighthouse-core/gather/gatherers/full-page-screenshot.js

```diff
@@ -45,13 +44,9 @@ class FullPageScreenshot extends Gatherer {
     await driver.sendCommand('Emulation.setDeviceMetricsOverride', {
       mobile: passContext.baseArtifacts.TestedAsMobileDevice,
       height,
-      screenHeight: height,
       width,
-      screenWidth: width,
-      deviceScaleFactor,
+      deviceScaleFactor: 1,
       scale: 1,
-      positionX: 0,
```

Comment:
removing all of this is necessary? IIRC you suggested I add all the parameters in the original PR.

Reply:
yeah i was being overly cautious back then. i looked and neither CDT nor PPTR uses these props.
i tested this heavily after these changes and feel good that they aren't important.